app/bundles/LeadBundle/Model/ListModel.php

```diff
@@ -411,6 +460,118 @@ public function getChoiceFields()
                 ],
                 'object' => 'lead',
             ],
+            'referer' => [
+                'label'      => $this->translator->trans('mautic.lead.list.filter.referer'),
+                'properties' => [
+                    'type' => 'text',
+                ],
+                'operators' => [
+                    'include' => [
+                        '=',
+                        'like',
+                    ],
+                ],
+                'object' => 'lead',
+            ],
+            'url_title' => [
+                'label'      => $this->translator->trans('mautic.lead.list.filter.url_title'),
+                'properties' => [
+                    'type' => 'text',
+                ],
+                'operators' => [
+                    'include' => [
+                        '=',
+                        'like',
+                    ],
+                ],
+                'object' => 'lead',
+            ],
+            'source' => [
+                'label'      => $this->translator->trans('mautic.lead.list.filter.source'),
+                'properties' => [
+                    'type' => 'text',
+                ],
+                'operators' => [
+                    'include' => [
+                        '=',
+                        'like',
+                    ],
+                ],
+                'object' => 'lead',
+            ],
+            'notification' => [
+                'label'      => $this->translator->trans('mautic.lead.list.filter.notification'),
+                'properties' => [
+                    'type' => 'text',
+                ],
+                'operators' => [
+                    'include' => [
+                        'empty',
```

Comment:
Shouldn't it be boolean instead? Could there be any other values than boolean? If no - it definitely should be bollean))